When applying this PR on v1.18.0-alpha.2.252+89714227ffe0ec-dirty I get this crash in kube-proxy;
```
I0130 16:46:28.931309     260 shared_informer.go:212] Caches are synced for service config 
I0130 16:46:28.931323     260 shared_informer.go:212] Caches are synced for endpoints config 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x14cff1c]

goroutine 63 [running]:
k8s.io/kubernetes/pkg/proxy/ipvs.(*Proxier).syncProxyRules(0xc00000b4a0)
        /home/uablrek/go/src/k8s.io/kubernetes-orig/_output/local/go/src/k8s.io/kubernetes/pkg/proxy/ipvs/proxier.go:1604 +0x636c
k8s.io/kubernetes/pkg/proxy/ipvs.(*Proxier).OnEndpointsSynced(0xc00000b4a0)
        /home/uablrek/go/src/k8s.io/kubernetes-orig/_output/local/go/src/k8s.io/kubernetes/pkg/proxy/ipvs/proxier.go:894 +0x62
k8s.io/kubernetes/pkg/proxy/config.(*EndpointsConfig).Run(0xc00038ba40, 0xc0000840c0)
        /home/uablrek/go/src/k8s.io/kubernetes-orig/_output/local/go/src/k8s.io/kubernetes/pkg/proxy/config/config.go:141 +0x108
created by k8s.io/kubernetes/cmd/kube-proxy/app.(*ProxyServer).Run
        /home/uablrek/go/src/k8s.io/kubernetes-orig/_output/local/go/src/k8s.io/kubernetes/cmd/kube-proxy/app/server.go:698 +0xb78
```
I have not investigated anything so I might have made some misstake, but just FYI.

--- a/go.mod
+++ b/go.mod
@@ -47,7 +47,7 @@ require (
 	github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
 	github.com/docker/go-connections v0.3.0
 	github.com/docker/go-units v0.4.0
-	github.com/docker/libnetwork v0.8.0-dev.2.0.20190624125649-f0e46a78ea34
+	github.com/docker/libnetwork v0.8.0-dev.2.0.20190925143933-c8a5fca4a652
 	github.com/elazarl/goproxy v0.0.0-20180725130230-947c36da3153
 	github.com/emicklei/go-restful v2.9.5+incompatible
 	github.com/euank/go-kmsg-parser v2.0.0+incompatible // indirect
@@ -250,7 +250,7 @@ replace (
 	github.com/docker/docker => github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
 	github.com/docker/go-connections => github.com/docker/go-connections v0.3.0
 	github.com/docker/go-units => github.com/docker/go-units v0.4.0
-	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20190624125649-f0e46a78ea34
+	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20190925143933-c8a5fca4a652
 	github.com/docker/spdystream => github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96
 	github.com/dustin/go-humanize => github.com/dustin/go-humanize v1.0.0
 	github.com/elazarl/goproxy => github.com/elazarl/goproxy v0.0.0-20180725130230-947c36da3153 // 947c36da3153 is the SHA for git tag v1.11

